docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md

```diff
@@ -293,6 +302,197 @@ The tablet leader is missing for more than 5 minutes for `$value` tablets in uni
+
+```expression
+(min_over_time((ybp_universe_replication_factor{universe_uuid='{{ $labels.universe_uuid }}'} - on(universe_uuid) count by(universe_uuid) (count by (universe_uuid, exported_instance) (follower_lag_ms{export_type="master_export", universe_uuid='{{ $labels.universe_uuid }}'})))[{{query_threshold }}s:]) > 0 or (max by(universe_uuid) (follower_lag_ms{export_type="master_export", universe_uuid='{{ $labels.universe_uuid }}'}) {{ query_condition }} ({{ query_threshold }} * 1000)))
+```
+
+### New YSQL Tables Added
+
+New YSQL tables are added to the source universe `'$universe_name'` in the database with an existing xCluster configuration, but not added to the xCluster replication.
+
+```expression
+((count by (namespace_name, universe_uuid)(count by(namespace_name, table_id, universe_uuid)(rocksdb_current_version_sst_files_size{universe_uuid="__universeUuid__",table_type="PGSQL_TABLE_TYPE"}))) - count by(namespace_name, universe_uuid)(count by(namespace_name, universe_uuid, table_id)(async_replication_sent_lag_micros{universe_uuid="__universeUuid__",table_type="PGSQL_TABLE_TYPE"}))) {{ query_condition }} {{ query_threshold }}
+```
+
+### PITR Config Failure
+
+Last Snapshot task for universe `'$universe_name'` failed. To retry, check PITR Configuration task result for more details.
+
+```expression
+min(ybp_pitr_config_status{universe_uuid = "__universeUuid__"}) {{ query_condition }} 1
+```
+
+### Private access key permission status
+
+Invalid Permissions of private access key file for universe `'$universe_name'`. You need to check YugabyteDB Anywhere logs for details or contact {{% support-platform %}}.
+
+```expression
+last_over_time(ybp_universe_private_access_key_status{universe_uuid = "__universeUuid__"}[1d]) {{ query_condition }} 1
+```
+
+### Replication Lag
+
+Average replication lag for universe `'$universe_name'` is above `$threshold` milliseconds. Current value is `$value` milliseconds.
+
+```expression
+max by (universe_uuid) (avg_over_time(async_replication_committed_lag_micros{universe_uuid="__universeUuid__"}[10m]) or avg_over_time(async_replication_sent_lag_micros{universe_uuid="__universeUuid__"}[10m])) / 1000 {{ query_condition }} {{ query_threshold }}
+```
+
+### SSH Key Rotation Failure
+
+Last SSH Key Rotation task for universe `'$universe_name'` failed. To retry, check SSH Key Rotation task result.
+
+```expression
+last_over_time(ybp_ssh_key_rotation_status{universe_uuid = "__universeUuid__"}[1d]) {{ query_condition }} 1
+```
+
+### SSH Key expiry
+
+SSH Key for universe `'$universe_name'` will expire in `$value` days.
+
+```expression
+ybp_universe_ssh_key_expiry_day{universe_uuid="__universeUuid__"} {{ query_condition }} {{ query_threshold }}
+```
+
+### Metric Collection Failure
+
+Failed to collect metric for universe `'$universe_name'`. You need to check YugabyteDB Anywhere logs for details or contact {{% support-platform %}}.
```

Comment:
just 1 metric? all metrics?

Reply:
I think the yml file has a typo. The summary has metric in it. Ill change it to metrics.